tmt/steps/provision/__init__.py

```diff
@@ -190,6 +190,8 @@ def _execute(
         except tmt.utils.RunError as exc:
             if exc.stdout and 'Please login as the user' in exc.stdout:
                 raise tmt.utils.GeneralError(f'Login to the guest failed.\n{exc.stdout}') from exc
+            if exc.stderr and 'executable file `/bin/bash` not found' in exc.stderr:
+                raise tmt.utils.GeneralError('Bash is required on the guest.') from exc
```

Comment:
I'd refer to `tmt.utils.DEFAULT_SHELL` instead of `/bin/bash` and `Bash`.

Reply:
yeah, did not see it